.github/workflows/e3sm-gh-pages.yml

```diff
@@ -12,28 +12,46 @@ on:
 concurrency:
   # Prevent 2+ copies of this workflow from running concurrently
   group: e3sm-docs-action
-  
+
 jobs:
-  Build-and-Deploy-docs:
+  e3sm-docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - 
+        name: Check out code
+        uses: actions/checkout@v4
         with:
           show-progress: false
-          fetch-depth: 0 # Needed, or else gh-pages won't be fetched, and push rejected
+          # We will need submodules when/if we build EAMxx docs here
```

Comment:
Oh?  Why is that?  EAMxx is not a submodule.

Reply:
EAMxx docs run some scripts that generates documentation for all input parameters, based on `namelist_defaults_scream.xml`. In doing so, it indirectly needs some python modules from CIME. Therefore, we do need at least the CIME submodule in order to build the eamxx docs.

If anyone thinks we should not require submodules just to build docs, we can find a workaround.

Reply:
Checking  out submodules noticeably slows down the model clone time.  That would be the only reason to do it.

Reply:
Yes, that's true.

@jgfouca I think we can do away with CIME in our buildnml & friends scripts if we roll our own version of `expect` (or copy/paste CIME's). I think that's the _only_ reason why we use a CIME module in the `.py` helper files.